pybamm/expression_tree/symbol.py

```diff
@@ -788,7 +824,7 @@ def has_symbol_of_classes(self, symbol_classes):
 
     def simplify(self, simplified_symbols=None, clear_domains=True):
         """ Simplify the expression tree. See :class:`pybamm.Simplification`. """
-        return pybamm.Simplification(simplified_symbols).simplify(self, clear_domains)
+        raise pybamm.ModelError(".simplify() is deprecated as it now has no effect")
```

Comment:
need to add a test for this line

Reply:
I'm sorry but what do you mean by test for this line? (Apologies if this is a foolish question but I'm new to open source programming)

Reply:
We can test that errors are raised when expected, for example like in this line
https://github.com/pybamm-team/PyBaMM/blob/develop/tests/unit/test_citations.py#L21

Reply:
Thank you @tinosulzer , I have now added the test.